packages/netlify-cms-core/src/components/App/Header.js

```diff
@@ -1,9 +1,8 @@
-/** @jsx jsx */
 import PropTypes from 'prop-types';
 import React from 'react';
 import ImmutablePropTypes from 'react-immutable-proptypes';
 import styled from '@emotion/styled';
-import { jsx, css } from '@emotion/core';
+import { css } from '@emotion/core';
```

Comment:
There are a bunch of these - I removed the jsx pragma in favor of the emotion babel plugin.